bundles/org.openhab.binding.mercedesme/src/main/java/org/openhab/binding/mercedesme/internal/VehicleHandler.java

```diff
@@ -0,0 +1,576 @@
+                    getImageResources();
+                }
+                String key = command.toFullString() + "_" + config.get().vin;
+                String encodedImage = EMPTY;
+                if (imageStorage.get().containsKey(key)) {
+                    encodedImage = imageStorage.get().get(key);
+                    logger.trace("Image {} found in storage", key);
+                } else {
+                    logger.trace("Request Image {} ", key);
+                    encodedImage = getImage(command.toFullString());
+                    if (!encodedImage.equals(EMPTY)) {
+                        imageStorage.get().put(key, encodedImage);
+                    }
+                }
+                if (encodedImage != null && !EMPTY.equals(encodedImage)) {
+                    RawType image = new RawType(Base64.getDecoder().decode(encodedImage),
+                            MIME_PREFIX + config.get().format);
+                    updateState(new ChannelUID(thing.getUID(), GROUP_IMAGE, "image-data"), image);
+                } else {
+                    logger.debug("Image {} is empty", key);
+                }
+            }
+        } else if (channelUID.getIdWithoutGroup().equals("clear-cache") && command.equals(OnOffType.ON)) {
+            List<String> removals = new ArrayList<String>();
+            imageStorage.get().getKeys().forEach(entry -> {
+                if (entry.contains("_" + config.get().vin)) {
+                    removals.add(entry);
+                }
+            });
+            removals.forEach(entry -> {
+                imageStorage.get().remove(entry);
+            });
+            updateState(new ChannelUID(thing.getUID(), GROUP_IMAGE, "clear-cache"), OnOffType.OFF);
+            getImageResources();
+        }
+    }
+
+    @Override
+    public void initialize() {
+        config = Optional.of(getConfigAs(VehicleConfiguration.class));
+        Bridge bridge = getBridge();
+        if (bridge != null) {
+            updateStatus(ThingStatus.UNKNOWN);
+            BridgeHandler handler = bridge.getHandler();
+            if (handler != null) {
+                accountHandler = Optional.of((AccountHandler) handler);
+                startSchedule(config.get().refreshInterval);
+                if (!config.get().vin.equals(NOT_SET)) {
+                    imageStorage = Optional.of(storageService.getStorage(BINDING_ID + "_" + config.get().vin));
+                    if (!imageStorage.get().containsKey(EXT_IMG_RES + config.get().vin)) {
+                        getImageResources();
+                    }
+                    setImageOtions();
+                }
+                updateState(new ChannelUID(thing.getUID(), GROUP_IMAGE, "clear-cache"), OnOffType.OFF);
+            } else {
+                String textKey = Constants.PREFIX + "vehicle" + Constants.STATUS_BRIDGEHANDLER_MISSING;
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED, textKey);
```

Comment:
This is not possible as a bridge is guaranteed to be initialized before its children. Therefore BRIDGE_UNINITIALIZED should only be set by the framework, as children will never be initialized while bridge is not.

Reply:
Checked `ThingStatusDetail`Constants and `HANDLER_MISSING_ERROR` should reflect it the right way